rfc/2022/2022-08-23_CIT_hostnames.md

```diff
@@ -0,0 +1,246 @@
+# Hostname Transition Release Plan
+**Cloud Isolation Team**
+  - Andrea Singh (Engineering)
+  - Alan Werner (Product)
+  - Patrick Vinograd (Contributor)
+
+Who is this document for?
+  - Platform and VFS development teams that work on the front-end of vets-website or the Mobile Flagship app.
+  - Platform and VFS development teams that work on vets-api.
+  - Lighthouse development teams that are involved in the migration of the Lighthouse API gateway. 
+
+## **Background**
+Vets-api and Lighthouse currently share the same API endpoint: api.va.gov. In the near future Lighthouse will assume exclusive control of api.va.gov and consumers of VA.gov APIs will need to transition to using a new API hostname: platform-api.va.gov. The goal of the Cloud Isolation Team is to enable this separation of concerns.
+
+## **Motivation**
+
+The product goals and technical infrastructures of Lighthouse and VA.gov have diverged over time. Sharing the api.va.gov hostname is no longer feasible. In particular, Lighthouse wants to re-architect to use an Apigee API gateway hosted in Google Cloud (GCP). There's no reason for all VA.gov traffic to route through this gateway or through GCP. Because Lighthouse has a large number of API consumers, most of whom are external to VA, they have priority for continuing to use api.va.gov. VA.gov will transition to a new API hostname.
```

Comment:
It's worth noting that the sharing of cloud infrastructure also complicates configuration changes, debugging and error management. If we make changes to revproxy or forward proxy config, we have to coordinate that these changes have not disrupted traffic from other teams. Similarly, when an error is occurring the traffic is difficult to isolate to a single team or application, causing additional workload and delaying resolution.

Reply:
Thanks, I expanded on the Motivation section to include those details